source/css/_common/components/highlight/highlight.styl

```diff
@@ -8,6 +8,7 @@ $code-block {
   overflow: auto;
   margin: 20px 0;
   padding: 0;
+  font-size: $font-size-base;
```

Comment:
Font-size of `$code-block` should be 1em. Using `font-size: $font-size-base;` will break it. See: https://github.com/theme-next/hexo-theme-next/issues/687

Reply:
Same as previous: if code size will be same as 1em, it will be ugly look:

**Nice look (was before)**:
![image](https://user-images.githubusercontent.com/16944225/56729087-a08cb300-6754-11e9-83cc-ff81ae5cdee5.png)

**Ugly look**:
![image](https://user-images.githubusercontent.com/16944225/56729103-b26e5600-6754-11e9-8901-8dd6f2e1b5e2.png)

Reply:
And it will not affect #687 issue because `rem` usage. So, `code` and `fa` sizes will be a little smaller, but with increasing fonts will save relative sizes. Try to increase global font size and see.

Reply:
See Github code sizes:

**1em (as u suggest)**:
![image](https://user-images.githubusercontent.com/16944225/56729310-3fb1aa80-6755-11e9-8825-3721f08ad837.png)

**12px (as is)**:
![image](https://user-images.githubusercontent.com/16944225/56729345-548e3e00-6755-11e9-94d8-3cd0ede66a0b.png)